src/net.cpp

```diff
@@ -2495,8 +2495,10 @@ bool CConnman::InitBinds(const Options& options)
     for (const auto& addrBind : options.vWhiteBinds) {
         fBound |= Bind(addrBind.m_service, (BF_EXPLICIT | BF_REPORT_ERROR), addrBind.m_flags);
     }
+    bool bound_onion{false};
     for (const auto& addr_bind : options.onion_binds) {
-        fBound |= Bind(addr_bind, BF_EXPLICIT | BF_DONT_ADVERTISE, NetPermissionFlags::None);
+        bound_onion |= Bind(addr_bind, BF_EXPLICIT | BF_REPORT_ERROR | BF_DONT_ADVERTISE, NetPermissionFlags::None);
+        fBound |= bound_onion;
```

Comment:
Please move the `fBound |= bound_onion;` out of the loop. No point in doing it every iteration, and it (imo) makes it sightly easier to reason about.

Reply:
Not relevant with the latest push.